src/learner.cc

```diff
@@ -493,6 +493,14 @@ class LearnerConfiguration : public Learner {
 
     this->ConfigureMetrics(args);
 
+    xgboost::processing::ProcessorLoader loader;
+    processor_instance = loader.load("dummy");
```

Comment:
Hi @trivialfis, to enable secureboost, we need to initiate this instance of processor, which will take some parameters, currently they are hard-coded. We wonder if you have some suggestion on where to inject the parameters? I thought about via training params, but kind of feeling awkward as it is only related to a particular federated scheme.